packages/react-components/src/components/VideoTile.tsx

```diff
@@ -156,32 +156,34 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           </Stack>
         )}
 
-        <Stack
-          horizontal
-          className={mergeStyles(
-            isVideoRendered ? videoHint : disabledVideoHint,
-            // when video is being rendered, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
-          )}
-        >
-          <Stack>
-            {displayName && (
-              <Text
-                className={mergeStyles(displayNameStyle, {
-                  color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
-                })}
-              >
-                {displayName}
-              </Text>
+        {(displayName || showMuteIndicator) && (
+          <Stack
+            horizontal
+            className={mergeStyles(
+              isVideoRendered ? videoHint : disabledVideoHint,
+              // when video is being rendered, the displayName has a grey-ish background, so no use of theme
+              { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
+              styles?.displayNameContainer
             )}
+          >
+            <Stack>
+              {displayName && (
+                <Text
+                  className={mergeStyles(displayNameStyle, {
+                    color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
```

Comment:
This logic is duplicated here and on line 165.
Actually, do you need this here at all? It should be inherited from the surrounding Stack, no?

Reply:
If I remove this here, I end up with display name color being almost unreadable
![image](https://user-images.githubusercontent.com/82416644/131754647-29e08118-677b-41ff-98e2-751af2178562.png)

And if I remove this color in line 165, it's the mic icon that is almost unreadable